lib/zipkin-tracer/trace_wrapper.rb

```diff
@@ -0,0 +1,18 @@
+module ZipkinTracer
+  class TraceWrapper
+    def self.wrap_in_custom_span(config, span_name, span_kind: Trace::Span::Kind::SERVER, app: nil)
```

Comment:
I think the span kind will be always `SERVER`, maybe it doesn't need to be passed as an argument... 🤔

Reply:
That's my thinking too but allowing for it to be passed gives the extra flexibility at no cost.
We've already seen people doing wonky things with the library so might be someone will come up with a specific use case where they need to customize the kind.